src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+public class Formatter {
+	
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+	
+    /** Formats a list of strings as a viewable indexed list. */
+    static String getIndexedListForViewing(List<String> listItems) {
+        final StringBuilder formatted = new StringBuilder();
+        int displayIndex = 0 + TextUi.DISPLAYED_INDEX_OFFSET;
+        for (String listItem : listItems) {
+            formatted.append(getIndexedListItem(displayIndex, listItem)).append("\n");
+            displayIndex++;
+        }
+        return formatted.toString();
+    }
+
+    /**
+     * Formats a string as a viewable indexed list item.
+     *
+     * @param visibleIndex visible index for this listing
+     */
+    private static String getIndexedListItem(int visibleIndex, String listItem) {
```

Comment:
Perhaps more formatting-related stuff (e.g. `DIVIDER`) can be moved to the `Formatter`? May be even things like this should be moved to the Formatter? Note that it may not always be clear if something belongs in the Formatter class or the TextUi class. It depends on how you define the responsibility of the Formatter class.